src/Styleguide/Elements/Flex.tsx

```diff
@@ -36,7 +36,7 @@ import { ClassAttributes, HTMLAttributes } from "react"
 
 const flexGrow = style({
   prop: "flexGrow",
-  numberToPx: false,
+  // numberToPx: false,
```

Comment:
You can just delete this. I think it was false by default even before 3.0 anyway 🤷‍♂️